### ISSUE TITLE
SQLiteException: near "on": syntax error

### ISSUE THREAD

**Issue by Pear0** [requery/requery#387]
My version of requery is 1.1.0.

I get the following error when Requery initializes `android.database.sqlite.SQLiteException: near "on": syntax error (code 1): , while compiling: create unique index on FeedSubscription (feedId)`.

The following statements are executed:
```sql
create table FeedSubscription (id integer primary key autoincrement not null, accentColor integer not null, description varchar(255), feedId varchar(255) unique, name varchar(255), photoUrl varchar(255), primaryColor integer not null, subscribed boolean not null)
create unique index on FeedSubscription (feedId)
```

If I manually provide a value for `Index#value` I get no error and the `CREATE INDEX` statement is valid:
```sql
create unique index feed_id on FeedSubscription (feedId)
```

The default value `""` for `Index#value` appears to be used which leads to a malformed sql statement. This differs from the Javadoc for `Index#value`.

> If empty a generic name will be created for the index.

For reference, my model class:
```java
@Entity
public interface FeedSubscription extends Parcelable, Persistable {

    @Key @Generated
    int getId();

    @Index
    //@Index("feed_id") 
    @Column(unique = true)
    String getFeedId();

    void setFeedId(String id);

    String getName();

    void setName(String name);

    String getDescription();

    void setDescription(String description);

    String getPhotoUrl();

    void setPhotoUrl(String photoUrl);

    int getPrimaryColor();

    void setPrimaryColor(int color);

    int getAccentColor();

    void setAccentColor(int color);

    boolean isSubscribed();

    void setSubscribed(boolean subscribed);

}
```

**Comment by npurushe** [comment]
Thanks for the detailed bug report, will be fixed in the next update thanks

**Comment by npurushe** [comment]
Thanks for reporting this fix is available in 1.1.1
